### PR TITLE
Adds systems in Kelowna, Canada

### PR DESCRIPTION
**Pull request by mjarrett**
Adds 4 scooter systems in Kelowna, Canada. 

From: https://www.kelowna.ca/roads-transportation/active-transportation/cycling/bikeshare-permit-program

### CODE REVIEW

**Comment by heidiguenin** [comment]
Thanks @mjarrett! Could you revise the System IDs to be "system_city"? We'll be working on guidance for naming in 2020 but at least want to stop adding systems that just have numbers for system IDs for now.

**Comment by mjarrett** [comment]
Makes sense! I was just pulling from the system_information.json.

**Comment by heidiguenin** [comment]
@mjarrett Ok my bad - I was getting ahead of myself. For now that's the right thing to do. We ask that operators make sure that the system_id is globally unique (even between different systems) and is intended to remain the same over the life of the system. Which several of these don't seem to adhere to, but your first approach was correct. 

In the future we'll be moving systems.csv into a different repo and will start working with operators to clean up these issues, but for now we should stick with your initial update.

File: systems.csv [hunk 1]
```diff
@@ -22,11 +22,15 @@ BR,Bike Itaú - Riviera,"Riviera, BR",rivi_bike,https://www.rivieradesaolourenco
 BR,Bike VV,"Vila Velha, BR",bike_vv,https://www.bikevv.com.br/,https://vilavelha.publicbikesystem.net/ube/gbfs/v1/
 CA,Bike Share Toronto,"Toronto, ON",bike_share_toronto,https://www.bikesharetoronto.com/,https://tor.publicbikesystem.net/ube/gbfs/v1/
 CA,BIXI-Montreal,"Montreal, QC",Bixi_MTL,https://www.bixi.com/,https://api-core.bixi.com/gbfs/gbfs.json
+CA,Bunny,"Kelowna, BC",bunny_kelowna,http://bunnyscooters.com/,https://bunny-api.joyridecity.bike//gbfs/en/gbfs.json
```
Comment (bddq) [inline systems.csv:25]: You can remove double slash here.

File: systems.csv [hunk 2]
```diff
@@ -22,11 +22,15 @@ BR,Bike Itaú - Riviera,"Riviera, BR",rivi_bike,https://www.rivieradesaolourenco
...
 CA,HOPR Ottawa,"Ottawa, ON",6,https://gohopr.com/velogo/,https://gbfs.hopr.city/api/gbfs/6/
 CA,HOPR Vancouver,"Vancouver, BC",13,https://www.mobibikes.ca/,https://gbfs.hopr.city/api/gbfs/13/
 CA,Mobi Bike Share,"Vancouver, BC",mobi_bikes,https://www.mobibikes.ca/,https://vancouver-gbfs.smoove.pro/gbfs/gbfs.json
 CA,nextbike Canada,"Victoria, CA",nextbike_ca,https://canada.nextbike.net/,https://gbfs.nextbike.net/maps/gbfs/v1/nextbike_ca/gbfs.json
+CA,ROLL Technologies Inc,"Kelowna, BC",roll_kelowna,https://www.rollscooters.com/,https://main.rollapi.com/v1/gbfs/gbfs.json
+CA,Segway OGO,"Kelowna, BC",Segway ogo_kelowna,https://www.ogoscooters.com/,https://us-dashboard.segway.pt/gbfs/partners/v1/Segway%20OGO/gbfs.json
 CA,Sobi Hamilton,Hamilton Ontario,sobi_hamilton,https://hamilton.socialbicycles.com/,https://hamilton.socialbicycles.com/opendata/gbfs.json
+CA,Zip,"Kelowna, BC",zip_kelowna,https://www.ridezip.co/,http://zip-api.joyridecity.bike/gbfs/en/gbfs.json
```
Comment (bddq) [inline systems.csv:33]: The GBFS server is also accessible through HTTPS protocol. Prefer HTTPS 👍 .

**Review by bddq — CHANGES_REQUESTED** [review]

File: systems.csv [hunk 3]
```diff
@@ -22,11 +22,15 @@ BR,Bike Itaú - Riviera,"Riviera, BR",rivi_bike,https://www.rivieradesaolourenco
...
 CA,Bike Share Toronto,"Toronto, ON",bike_share_toronto,https://www.bikesharetoronto.com/,https://tor.publicbikesystem.net/ube/gbfs/v1/
 CA,BIXI-Montreal,"Montreal, QC",Bixi_MTL,https://www.bixi.com/,https://api-core.bixi.com/gbfs/gbfs.json
+CA,Bunny,"Kelowna, BC",bunny_kelowna,http://bunnyscooters.com/,https://bunny-api.joyridecity.bike//gbfs/en/gbfs.json
 CA,HOPR Ottawa,"Ottawa, ON",6,https://gohopr.com/velogo/,https://gbfs.hopr.city/api/gbfs/6/
 CA,HOPR Vancouver,"Vancouver, BC",13,https://www.mobibikes.ca/,https://gbfs.hopr.city/api/gbfs/13/
 CA,Mobi Bike Share,"Vancouver, BC",mobi_bikes,https://www.mobibikes.ca/,https://vancouver-gbfs.smoove.pro/gbfs/gbfs.json
 CA,nextbike Canada,"Victoria, CA",nextbike_ca,https://canada.nextbike.net/,https://gbfs.nextbike.net/maps/gbfs/v1/nextbike_ca/gbfs.json
+CA,ROLL Technologies Inc,"Kelowna, BC",roll_kelowna,https://www.rollscooters.com/,https://main.rollapi.com/v1/gbfs/gbfs.json
+CA,Segway OGO,"Kelowna, BC",Segway ogo_kelowna,https://www.ogoscooters.com/,https://us-dashboard.segway.pt/gbfs/partners/v1/Segway%20OGO/gbfs.json
```
Comment (heidiguenin) [inline systems.csv:31]:
```suggestion
CA,Segway OGO,"Kelowna, BC",Segway OGO,https://www.ogoscooters.com/,https://us-dashboard.segway.pt/gbfs/partners/v1/Segway%20OGO/gbfs.json
```

Initially I had asked for this to be revised, but see comment in thread about needing to clean-up ids later.

File: systems.csv (hunk 2 above)
Comment (heidiguenin) [inline systems.csv:33]:
```suggestion
CA,Zip,"Kelowna, BC",Zip,https://www.ridezip.co/,http://zip-api.joyridecity.bike/gbfs/en/gbfs.json
```
Initially I had asked for this to be revised, but see comment in thread about needing to clean-up ids later.

File: systems.csv (hunk 1 above)
Comment (heidiguenin) [inline systems.csv:25]:
```suggestion
CA,Bunny,"Kelowna, BC",Bunny,http://bunnyscooters.com/,https://bunny-api.joyridecity.bike/gbfs/en/gbfs.json
```
Initially I had asked for this to be revised, but see comment in thread about needing to clean-up ids later.

File: systems.csv [hunk 4]
```diff
@@ -22,11 +22,15 @@ BR,Bike Itaú - Riviera,"Riviera, BR",rivi_bike,https://www.rivieradesaolourenco
 BR,Bike VV,"Vila Velha, BR",bike_vv,https://www.bikevv.com.br/,https://vilavelha.publicbikesystem.net/ube/gbfs/v1/
 CA,Bike Share Toronto,"Toronto, ON",bike_share_toronto,https://www.bikesharetoronto.com/,https://tor.publicbikesystem.net/ube/gbfs/v1/
 CA,BIXI-Montreal,"Montreal, QC",Bixi_MTL,https://www.bixi.com/,https://api-core.bixi.com/gbfs/gbfs.json
+CA,Bunny,"Kelowna, BC",bunny_kelowna,http://bunnyscooters.com/,https://bunny-api.joyridecity.bike//gbfs/en/gbfs.json
 CA,HOPR Ottawa,"Ottawa, ON",6,https://gohopr.com/velogo/,https://gbfs.hopr.city/api/gbfs/6/
 CA,HOPR Vancouver,"Vancouver, BC",13,https://www.mobibikes.ca/,https://gbfs.hopr.city/api/gbfs/13/
 CA,Mobi Bike Share,"Vancouver, BC",mobi_bikes,https://www.mobibikes.ca/,https://vancouver-gbfs.smoove.pro/gbfs/gbfs.json
 CA,nextbike Canada,"Victoria, CA",nextbike_ca,https://canada.nextbike.net/,https://gbfs.nextbike.net/maps/gbfs/v1/nextbike_ca/gbfs.json
+CA,ROLL Technologies Inc,"Kelowna, BC",roll_kelowna,https://www.rollscooters.com/,https://main.rollapi.com/v1/gbfs/gbfs.json
```
Comment (heidiguenin) [inline systems.csv:30]:
```suggestion
CA,ROLL Technologies Inc,"Kelowna, BC",0,https://www.rollscooters.com/,https://main.rollapi.com/v1/gbfs/gbfs.json
```
Initially I had asked for this to be revised, but see comment in thread about needing to clean-up ids later.

**Comment by heidiguenin** [comment]
@mjarrett Can you go in and accept the changes I made to revise back to systemID from system_information.json and @bixcorp's revisions?

File: systems.csv (hunk 4 above)
Comment (jcn) [reply systems.csv:30]: It also appears that this API is throwing errors on at least a few of its pages sporadically
Screen Shot 2020-02-17 at 12 58 39 AM
.

Comment (RollTechInc) [reply systems.csv:30]: Hi, @jcn and @heidiguenin . This is Roll Technologies Inc. Our GBFS apis are recently changed to under 

> https://data.rollapi.com/

Sorry for your inconvenience.

**Comment by mplsmitch** [comment]
Closing this PR because it's no longer relevant. I've gone through all of these feeds and none of them are active at this point. The Kelowna website has not been updated to reflect this. We've got a new feed for Roll. We'll keep an eye on the others going forward.

**Pull request closed without merging**